src/main/charts/confluence/templates/configmap-fluentd.yaml

```diff
@@ -6,13 +6,19 @@ metadata:
   labels:
   {{- include "confluence.labels" . | nindent 4 }}
 data:
+  {{ if .Values.fluentd.customConfigFile }}
```

Comment:
@nanux Its only one or the other , in the values file I have
``` defaultConfig: false
  # -- True if a custom config should be used for fluentd
  customConfigFile: false
  # -- Custom fluent.conf file
  ```

Reply:
As they are eventually translated to two distinct `if` statements it is currently valid to provide:
```
defaultConfig: true
customConfigFile: true
```
and it will add the default config and the custom config after it. Is that the planned behaviour? Would be possible to replace it with single optional value `fluentd.customConfig: `?

pseudocode:
```
if fluentd.customConfig:
    fluentd.customConfig
else
     fluent.conf: |
    <source>
      @type tail
....
end